style/app.scss

```diff
@@ -669,6 +673,11 @@ div.sidebar.collapsed {
 }
 
 .spotlight {
+  fill: #ffd9b7;
```

Comment:
Great, please make a bit more yellow, and lighter (less saturated)

Reply:
I am afraid that by making it more yellow the contrast between the spotlight and the course nodes in the "Mathematics and Statistics" category might not be sufficient, but let me know if it works.